src/main/groovy/com/palantir/gradle/javadist/DistributionExtension.groovy

```diff
@@ -19,7 +19,8 @@ class DistributionExtension {
 
     private static final List<String> requiredJvmOpts = [
         '-Djava.security.egd=file:/dev/./urandom',
-        '-Djava.io.tmpdir=var/data/tmp'
+        '-Djava.io.tmpdir=var/data/tmp',
+        '-XX:+PerfDisableSharedMem'
```

Comment:
it's in the commit message, but I can add it here too. good idea.

On Fri, Jul 15, 2016 at 11:52 AM Mark Elliot <notifications@github.com>
wrote:

> In
> src/main/groovy/com/palantir/gradle/javadist/DistributionExtension.groovy
> <https://github.com/palantir/gradle-java-distribution/pull/93#discussion_r71023397>
> :
>
> > @@ -19,7 +19,8 @@ class DistributionExtension {
> >
> >      private static final List<String> requiredJvmOpts = [
> >          '-Djava.security.egd=file:/dev/./urandom',
> > -        '-Djava.io.tmpdir=var/data/tmp'
> > +        '-Djava.io.tmpdir=var/data/tmp',
> > +        '-XX:+PerfDisableSharedMem'
>
> can we add a comment linking to the article describing impact of this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/gradle-java-distribution/pull/93/files/70194f64a199df66a9914bff1aa7b2387d05241d#r71023397>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwWHEhVwJguYFENi8JB_oPg_GuGopks5qV9a9gaJpZM4JNqj2>
> .
>